career-readiness.md

```diff
@@ -8,15 +8,15 @@ Creating a technical portfolio website is a big undertaking. Portfolios are almo
 
 Wire frames are used to outline the basic structure of your site. Wire frames should be vague enough to give you flexibility in your development process but structured enough to ensure all appropriate elements are present. 
 
-Submit the link to your wire frames:
+Submit the link to your wire frames: https://www.figma.com/file/nBenpfchFiXVljIU4m0dw6/Portfolio?type=design&node-id=0%3A1&mode=design&t=0Liek3YfLjrY7mV0-1
```

Comment:
I suggest adding a bit more to these wireframes than just a heading and background colors

Reply:
been slowly working on this but i got you.